test/docker/docker-compose.yml

```diff
@@ -81,3 +81,30 @@ services:
       DRAND_PUBLIC_ADDRESS: "drand_4:8480"
       DRAND_SHARE_SECRET: "thisisthesecretweshouldsettorundkgprocessonthenodes"
 
+  drand_client:
+    container_name: drand_client
+    image: docker.io/drandorg/go-drand-dev:latest
```

Comment:
will we need to set up a repo / publish this?
(there's some extra devops code that needs to exist somewhere to do that if so)
I guess the makefile build will make that image available under that name locally, so that may be enough.

I think the ask then is to add a readme in `test/docker` explaining what needs to happen to run these tests.

Reply:
@willscott it is done. Could you please check it?